### DISCUSSION TITLE
Plugin not found exception

### DISCUSSION THREAD

**Discussion by vaibhavswarnkar** [rabbitmq/rabbitmq-delayed-message-exchange#202] [General]
Every time I try installing this plugin, I just get the below error
```cli
rabbitmq-plugins enable rabbitmq_delayed_message_exchange
Enabling plugins on node rabbit@localhost:
rabbitmq_delayed_message_exchange
Error:
{:plugins_not_found, [:rabbitmq_delayed_message_exchange]}
```

I had installed RabbitMQ using homebrew and running on the 3.10.5 version. I had also tried installing it manually by downloading the tar and exploding the tar in the plugins directory and still fails with the same error.

Is this solution available in GA?? What could be the problem here?

**Comment by michaelklishin** [comment]
See [Troubleshooting](https://rabbitmq.com/plugins.html#troubleshooting) in the Plugins guide.
There is no need to extract the .ez file, RabbitMQ [does so on boot](https://rabbitmq.com/plugins.html#plugin-directories).

I'm not sure what tar that was as the plugin is distributed as a [single .ez binary](https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/releases/tag/3.10.2). My guess is that you've downloaded the source code tarball that GitHub produces instead of release artifacts.

**Reply by vaibhavswarnkar** [reply]
Thanks got it working. One suggestion, the installation document is not good, I'd suggest improving it and getting rid of the confusion.

**Reply by michaelklishin** [reply]
If you criticize something and "suggest improving it" you have to be more specific than "no good". What exactly confused you?
